Android start instructions with error in 0.29.1

Hi!
Thank you for your work, please tell me what i'm doing wrong
I'm getting this error:

`MainApplication.java:29: error: onCreate() in MainApplication cannot override onCreate() in Application
  protected void onCreate() {
                 ^
  attempting to assign weaker access privileges; was public`

this is my MainApplication file: 
`
package com.startmobile;

import android.app.Application;
import android.util.Log;

import com.facebook.react.ReactApplication;
import com.facebook.react.ReactInstanceManager;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;
import com.facebook.react.ReactActivity;
import io.realm.react.RealmReactPackage;
import com.airbnb.android.react.maps.MapsPackage;
import com.BV.LinearGradient.LinearGradientPackage;
import com.oblador.vectoricons.VectorIconsPackage;
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;
import com.microsoft.codepush.react.CodePush;
import com.smixx.fabric.FabricPackage;
import com.crashlytics.android.Crashlytics;
import io.fabric.sdk.android.Fabric;

import java.util.Arrays;
import java.util.List;

public class MainApplication extends Application implements ReactApplication {

  @Override
  protected void onCreate() {
    super.onCreate();
    Fabric.with(this, new Crashlytics());
  }

  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {

    @Override
    protected String getJSBundleFile() {
      return CodePush.getJSBundleFile();
    }

    @Override
    protected boolean getUseDeveloperSupport() {
      return BuildConfig.DEBUG;
    }

    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
        new MainReactPackage(),
        new RealmReactPackage(),
        new CodePush("**********", MainApplication.this, BuildConfig.DEBUG),
        new LinearGradientPackage(),
        new MapsPackage(),
        new FabricPackage(),
        new VectorIconsPackage()
      );
    }
  };

  @Override
  public ReactNativeHost getReactNativeHost() {
      return mReactNativeHost;
  }
}

`

--- a/README.md
+++ b/README.md
@@ -166,7 +166,7 @@ Make sure you also follow the steps described in [`Android`](#android).
   public class MainApplication extends Application implements ReactApplication {
   
   +   @Override
-  +   protected void onCreate() {
+  +   public void onCreate() {
   +       super.onCreate();
   +       Fabric.with(this, new Crashlytics());
   +   }
@@ -175,6 +175,8 @@ Make sure you also follow the steps described in [`Android`](#android).
   
   }
   ``` 
+
+* Note: the `onCreate` access privilege goes from `protected` to `public` from RN 0.28+
 
 * Edit your `AndroidManifest.xml` (in `android/app/src/main/`) to look like this. Make sure to enter your fabric API key after `android:value=`, you can find your key on your fabric organisation page.
 

